sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -1376,6 +1376,27 @@ def test_encoders_string_categories(input_dtype, category_dtype, array_type):
     assert_array_equal(X_trans, expected)
 
 
+def test_mixed_string_bytes_categoricals():
```

Comment:
Small typo:

```suggestion
def test_mixed_string_bytes_categoricals():
```

Reply:
What is the typo?

Reply:
Ah this is my mistake. There is no typo. 😅